enterprise/frontend/src/embedding-sdk/components/private/SdkGlobalStylesWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import type { HTMLAttributes } from "react";
+
+import { rootStyle } from "metabase/css/core/base.styled";
+import { defaultFontFiles } from "metabase/css/core/fonts.styled";
+import { useSelector } from "metabase/lib/redux";
+import { aceEditorStyles } from "metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.styled";
+import { getFontFiles } from "metabase/styled-components/selectors";
+import { saveDomImageStyles } from "metabase/visualizations/lib/save-chart-image";
+import type { FontFile } from "metabase-types/api";
+
+interface SdkContentWrapperProps {
+  baseUrl?: string;
+}
+
+export function SdkGlobalStylesWrapper({
+  baseUrl,
+  ...divProps
+}: SdkContentWrapperProps & HTMLAttributes<HTMLDivElement>) {
+  const fontFiles = useSelector(getFontFiles);
+
+  return (
+    <SdkGlobalStylesInner
+      baseUrl={baseUrl}
+      fontFiles={fontFiles}
+      {...divProps}
+    />
+  );
+}
+
+const SdkGlobalStylesInner = styled.div<
+  SdkContentWrapperProps & {
+    fontFiles: FontFile[] | null;
+  }
+>`
+  font-size: ${({ theme }) => theme.other.fontSize};
+
+  ${aceEditorStyles}
+  ${saveDomImageStyles}
+  ${rootStyle}
+
+  ${({ baseUrl }) => defaultFontFiles({ baseUrl })}
+
+  ${({ fontFiles }) =>
+    fontFiles?.map(
+      file => css`
+        @font-face {
+          font-family: "Custom";
+          src: url(${encodeURI(file.src)}) format("${file.fontFormat}");
+          font-weight: ${file.fontWeight};
+          font-style: normal;
+          font-display: swap;
+        }
```

Comment:
@font-face declarations are always global in nature. It is not scoped. We don't need to add them to `PublicComponentWrapper`